perun-engine/src/main/java/cz/metacentrum/perun/engine/scheduling/impl/PropagationMaintainerImpl.java

```diff
@@ -318,7 +318,7 @@ private void abortTask(Task task, TaskStatus status) {
 			}
 		} else {
 			log.error("[{}] Stale Task {} was not removed and not reported to dispatcher.", task.getId(), task);
-			log.error("  - This is nonsense - why we abort the Task taken from AllTasks but we can remove it from it ??");
+			log.warn("  - This is nonsense - why we abort the Task taken from AllTasks but we can remove it from it ??");
```

Comment:
This message is actually considered as continuation of previous log message, so it makes sens to keep them at same level.